### ISSUE TITLE
Degolburg - No Weapon or Armor Shop

### ISSUE THREAD

**Issue by chenagon** [idmarinas/lotgd-game#1496]
**Describe the bug**
In Degolburg and any city that I have traveled to, I only sometimes see the weapon or armor shop.  If I see it at all, I usually do not have enough gold to buy what I want so I go to the bank to withdraw gold.  After withdrawing gold and returning to the city I do not see the weapon or armor shop

**To Reproduce**
1. Logon
2. If see Weapon or Armor shop, go to bank to withdraw gold
3. Return to city, no weapon or armor shop

**Expected behavior**
I am assuming the Weapon and Armor shop are always visible in all cities.  Or at least always visible in Degolburg

**Screenshots**
![Degolburg](https://user-images.githubusercontent.com/18216907/229633304-28a3d58d-a74f-4418-9cd5-5e0d92966d4b.png)

**Desktop (please complete the following information):**

**Smartphone (please complete the following information):

**

**Additional context**

**Comment by idmarinas** [comment]
Hi:
This is not a bug, it is the normal operation, some services are only available in the capital, and others in your home city.

In this case the weapon and armor stores are only available in your home city.

Greetings and happy adventures

**Comment by chenagon** [comment]
Hello IDMarinas,

I encounter the same situation in Glorfindal, which I think should be my
home town as an elf.  Though I also noticed my race displays as "Humanoid",
is that supposed to change at some point?  If you want me to submit a
different bug report let me know, happy to provide more details if needed

Thanks,
Michael

**Comment by idmarinas** [comment]
If your race is Humanoid you no select race, may be by error in server.
Can select other race in the Capital Inn

The problem is this, if you see Humanoid as race.

**Comment by idmarinas** [comment]
I change your race to Elf.

If you not want this race you can change in Capital Inn o wait to defeated Dragon.

**Comment by chenagon** [comment]
Ah, thank you for your responses and the information

Some other questions I have come across:

   1. Since creating a character on Saturday April 1, I have received an
   email each day with the subject "LoGD Character Expiration" and with email
   body below -I have logged in each day. Is there something else I need to do
   to make sure my character does not expire?
      1. "One or more (Hency) of your characters in Legend of the Green
      Dragon at https://lotgd.infommo.es/ is about to expire.

      If you wish to keep this character, you should log on to him or her
      soon!"
   2. I purchased a Stallion, but now I think somehow I no longer have it.
   Not sure if I did something wrong or if there is a bug

Thank you for your hard work on this game, I am enjoying it very much!

Thanks,
Michael
